e2e-tests/playwright/e2e/guest-signin-happy-path.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { test, Page } from '@playwright/test';
+import { UIhelper } from '../utils/UIhelper';
+import { HomePage } from '../support/pages/HomePage';
+import { Common } from '../utils/Common';
+import { gpts } from '../support/testData/gpts';
+
+test.describe('Guest Signin Happy path', () => {
+  let page: Page;
+  let uiHelper: UIhelper;
+  let homePage: HomePage;
+  let common: Common;
+
+  test.beforeAll(async ({ browser }) => {
+    page = await browser.newPage();
+    uiHelper = new UIhelper(page);
+    homePage = new HomePage(page);
+    common = new Common(page);
+    await common.loginAsGuest();
+  });
+
+  test('Verify the Homepage renders with Search Bar, Quick Access and Starred Entities', async () => {
+    await uiHelper.verifyHeading('Welcome back!');
+    await uiHelper.openSidebar('Home');
+    await homePage.verifyQuickAccess('Developer Tools', 'Podman Desktop');
+  });
+
+  test('Verify Catalog page renders with no components', async () => {
+    await uiHelper.openSidebar('Catalog');
+    await uiHelper.verifyHeading('My Org Catalog');
+    await uiHelper.selectMuiBox('Kind', 'Component');
+    // Verificar se não há registros para exibir
+  });
+
+  test('Verify that all users in your Github Organization have been ingested into the Catalog Page', async () => {
+    await uiHelper.selectMuiBox('Kind', 'User');
+    await uiHelper.verifyRowsInTable([
+      'subhashkhileri',
+      'josephca',
+      'gustavolira',
+    ]);
+  });
+
+  test('Verify all 12 GPTs appear in the Create page', async () => {
+    await uiHelper.openSidebar('Create...');
+    await uiHelper.verifyHeading('Golden Path Templates');
```

Comment:
this heading has changed to Software Templates in https://github.com/janus-idp/backstage-showcase/commit/21eb11a467c83e1e6b679d096cb1bfc950f2d57d

See https://github.com/janus-idp/backstage-showcase/pull/889/files for one possible refactoring you could do (rename "gpt" to "template", etc.)

Reply:
```suggestion
    await uiHelper.verifyHeading('Software Templates');
```